en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-inbound-endpoint-example.md

```diff
@@ -0,0 +1,123 @@
+# Salesforce Inbound Endpoint Example
+
+In the real world, the Salesforce streaming Inbound Endpoint allows you to perform various operations on Salesforce streaming data via WSO2 EI.
+
+The Salesforce streaming API receives notifications based on the changes that happen to Salesforce data with respect to an SQQL (Salesforce Object Query Language) query you define, in a secured and scalable way. For more information, navigate to [Salesforce streaming documentation](https://developer.salesforce.com/docs/atlas.en-us.202.0.api_streaming.meta/api_streaming/quick_start_workbench.htm).
```

Comment:
Shall we add a link to the API if possible?

Reply:
Added Streaming API Developer Guide to the doc.